lightstep-tracer-common

```diff
@@ -1 +1 @@
-Subproject commit a2ac69205156c196ef5dbe7037fda7091580d1e9
+Subproject commit b1bdd09872f4e7d6439f613181997dedb1783458
```

Comment:
"git submodule update" doesn't do the right thing
but I did something similar, sure

ᐧ

On Thu, Sep 8, 2016 at 3:47 PM, spoons <notifications@github.com> wrote:

> In lightstep-tracer-common
> <https://github.com/lightstep/lightstep-tracer-go/pull/37#discussion_r78103148>
> :
>
> > @@ -1 +1 @@
> > -Subproject commit a2ac69205156c196ef5dbe7037fda7091580d1e9
> > +Subproject commit b1bdd09872f4e7d6439f613181997dedb1783458
>
> probably don't want this... maybe git submodule update?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lightstep/lightstep-tracer-go/pull/37/files/ceb1621105627997e0c141b9533888bd7bd9b96e#r78103148>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADdiiYdT5VZov1NrOgQOpcVbrSZUIB-jks5qoJCBgaJpZM4J4XE3>
> .
>